src/main/java/com/findmyrecycling/fmrenterprise/FindMyRecyclingController.java

```diff
@@ -21,66 +20,56 @@ public class FindMyRecyclingController {
     IFacilityService facilityService;
 
     @RequestMapping("/")
-    public String index(Model model) {
-        RecyclableMaterial recyclableMaterial = new RecyclableMaterial();
-        Facility facility = new Facility();
-        recyclableMaterial.setMaterialName("");
-        facility.setFacilityAddress("");
-        model.addAttribute(recyclableMaterial);
+    public String index() {
         return "index";
     }
 
-    @RequestMapping("/AddFacility.html")
+    @RequestMapping("/facility")
     public String addFacilityPage(Model model) {
         Facility facility = new Facility();
-        facility.setFacilityName("Bob's Junk-Yard");
-        facility.setFacilityPhotos("");
-        facility.setFacilityAddress("5764 Hills Drive");
-        facility.setFacilityId(110L);
-        facility.setMaterialId(50L);
-        model.addAttribute(facility);
-        return "AddFacility";
+        model.addAttribute("facility", facility);
+        return "facility";
     }
 
-    @RequestMapping("/saveFacility")
-    public String saveFacility(Facility facility) {
+    /**
+     * Create a facility.
+     * @param facility facility to be created.
+     * @return Redirects to the index.html page (home page).
+     */
```

Comment:
JoveDoc. These are missing everywhere in the project. Maybe a good reason?

Reply:
Good, descriptive JavaDoc.